src/networks/mainnet/mod.rs

```diff
@@ -170,6 +170,11 @@ pub(super) static DRAND_SCHEDULE: [DrandPoint<'static>; 2] = [
         height: SMOKE_HEIGHT,
         config: &DRAND_MAINNET,
     },
+    DrandPoint {
+        // height is TDB
```

Comment:
what does this mean?

Reply:
We need to update the right height to perform the upgrade later. It should be the Pineapple upgrade Epoch but it's not defined in forest yet.